src/openrct2/ride/VehicleSubpositionData.cpp

```diff
@@ -21932,80 +24395,116 @@ static constexpr const rct_vehicle_info_list *TrackVehicleInfoList_8BD228[] = {
+    &RightLane::TrackVehicleInfoLeftSBend0, &RightLane::TrackVehicleInfoLeftSBend1, &RightLane::TrackVehicleInfoLeftSBend2, &RightLane::TrackVehicleInfoLeftSBend3, // 38 s-bend left
+    &RightLane::TrackVehicleInfoRightSBend0, &RightLane::TrackVehicleInfoRightSBend1, &RightLane::TrackVehicleInfoRightSBend2, &RightLane::TrackVehicleInfoRightSBend3, // 39 s-bend right
+    &TrackVehicleInfo_8D44BE, &TrackVehicleInfo_8D4FA6, &TrackVehicleInfo_8D5A8E, &TrackVehicleInfo_8D6576, // 40 left loop
+    &TrackVehicleInfo_8D705E, &TrackVehicleInfo_8D7B46, &TrackVehicleInfo_8D862E, &TrackVehicleInfo_8D9116, // 41 right loop
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles3, // 42 small left
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles3, // 43 small right
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles3, // 44 banked small left
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles3, // 45 banked small right
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegUp3, // 46 up small left
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegUp3, // 47 up small right
+    &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown0, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown1, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown2, &RightLane::TrackVehicleInfoLeftQuarterTurn3Tiles25DegDown3, // 48 down small left
+    &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown0, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown1, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown2, &RightLane::TrackVehicleInfoRightQuarterTurn3Tiles25DegDown3, // 49 down small right
+    &TrackVehicleInfo_903DE1, &TrackVehicleInfo_903F0C, &TrackVehicleInfo_904040, &TrackVehicleInfo_904162, // 50 tiny left
+    &TrackVehicleInfo_904284, &TrackVehicleInfo_9042F2, &TrackVehicleInfo_904369, &TrackVehicleInfo_9043E9, // 51 tiny right
```

Comment:
Adding comments about the track element type is a big improvement, but could you use the definitions from the enum in Track.h (line 226 onwards)? E.g. `TRACK_ELEM_RIGHT_QUARTER_TURN_1_TILE`.